index.md

```diff
@@ -152,7 +152,7 @@ This block displays the date and links to Google Calendar.
 {% endcomment %}
 {% if page.humandate %}
 <p id="when">
-  <strong>When:</strong>
+  <strong>何日:</strong>
```

Comment:
```suggestion
  <strong>日程:</strong>
```

Reply:
Thanks, I wasn't sure about this one. I've made the same (changes)[https://github.com/swcarpentry-ja/workshop-template-ja/commit/4fc477065018452300cfffa2090bb3a3d22b331e) in the template repo.